src/api-reference/request/v4.response-codes.markdown

```diff
@@ -5,23 +5,21 @@ layout: reference
 
 {% include prerelease.html %}
 
-If a new or existing project please use version [3.0](./v3.request.html) of this API.
-
 HTTP Status Code|Response Body|Description|Wrong input example
```

Comment:
Do we need the Wrong Input examples? They make the table require a sideways scroll bar, which is a pain to work with. If they are providing value to developers, we can leave them in, but if not I'd rather remove them.

Reply:
@laurentlhermine  what do you think about maintaining or removing thos Wrong input examples?

Reply:
remove column